### PR TITLE
[GEARPUMP-233]Add hbase-example

### PR DESCRIPTION
**Pull request by Roshanson**

### CODE REVIEW

File: examples/streaming/hbase/src/main/scala/org/apache/gearpump/streaming/hbase/HBaseConn.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  def application(config: ParseResult, system: ActorSystem): StreamApplication = {
+    implicit val actorSystem = system
+
+    val splitNum = config.getInt("splitNum")
+    val sinkNum = config.getInt("sinkNum")
+
+    val split = new Split
+    val sourceProcessor = DataSourceProcessor(split, splitNum, "Split")
+    val sink = new Sink
```
Comment (huafengw) [inline examples/streaming/hbase/src/main/scala/org/apache/gearpump/streaming/hbase/HBaseConn.scala]: HBaseSink is already a DataSink so you don't need to implement this one.

**Comment by manuzhang** [comment]
please file a [JIRA issue](http://issues.apache.org/jira/browse/GEARPUMP) for this

**Comment by manuzhang** [comment]
please change the commit log to the form `[GEARPUMP-XXX] XYZ`

**Comment by manuzhang** [comment]
@huafengw what do you think of this ?

**Comment by huafengw** [comment]
+1

**Pull request closed without merging**